### DISCUSSION TITLE
Help using grafana bar gauge

### DISCUSSION THREAD

**Discussion by Bordocklius** [grafana/grafana-json-datasource#86] [Q&A] [answered]
Hi guys, i've got a bit of a problem. I managed to get the plugin up and running and even managed to make a few simple graphs, but i am a bit stuck on how to use the bar gauge with the plugin, as it isn't working as i thought it would.

![afbeelding](https://user-images.githubusercontent.com/58853886/112320594-711fb180-8caf-11eb-919e-227648794385.png)

This is the data i am currently getting from my API, which is getting it from Azure table storage, with some stuff scratched out. 
I am trying to chart the amount of each carepath type, so i add those fields to the query: 
![afbeelding](https://user-images.githubusercontent.com/58853886/112319730-a37cdf00-8cae-11eb-9735-809c469466ad.png)

Now the chart looks like this: 
![afbeelding](https://user-images.githubusercontent.com/58853886/112319898-cc04d900-8cae-11eb-8e0d-55126bcae51b.png)

I was wondering how i am supposed to get the type as the label for the bar. Any idea how i can get this to work?
Thanks in advance!

**Comment by marcusolsson** [comment, marked as answer]
The current Bar chart expects a query result with multiple time series where each bar represents a time series.

To support this, the JSON data source would have to support splitting the query response into multiple series, probably based on the value of a property in the response. There's an open feature request (https://github.com/marcusolsson/grafana-json-datasource/issues/36) to support this, so please vote for it if this is important to you.

**Reply by Bordocklius** [reply]
Alright, thanks for the reply, i'll be shure to vote for it.
